motile/track_graph.py

```diff
@@ -1,10 +1,13 @@
+from __future__ import annotations
+
 import logging
-import networkx as nx
+from typing import Any
+from networkx.classes import DiGraph
 
 logger = logging.getLogger(__name__)
 
 
-class TrackGraph(nx.DiGraph):
+class TrackGraph(DiGraph):  # type: ignore [misc]
```

Comment:
this type ignore is because DiGraph is untyped... and we're inheriting from it, which is an error in mypy strict mode

Reply:
`TrackGraph` no longer inherits from `DiGraph`, so that's fine.